provider/vdpa_sim_utils.py

```diff
@@ -159,3 +159,32 @@ def add_dev(self, name, mac):
             raise VDPABlkNetSimulatorError(
                 "vdpa dev add %s failed:%s" % (name, str(e)))
         return dev
+
+
+class VirtioVdpaNetSimulatorTest(VDPABlkNetSimulatorTest):
+    def __init__(self):
+        super(VirtioVdpaNetSimulatorTest, self).__init__()
+        self._modules = ['vdpa', 'virtio-vdpa', 'vdpa_sim', 'vdpa-sim-net']
+
+    def add_dev(self, name, mac):
+        """
+        Add vDPA net device
+        :param name: device name
+        :param mac: MAC address
+        :return : host device name ,eg. eth0
+        """
+
+        cmd = "vdpa dev add name %s mgmtdev vdpasim_net mac %s" % (name, mac)
+        process.run(cmd, shell=True)
+        cmd = "vdpa dev list -jp %s" % name
+        process.run(cmd, shell=True)
```

Comment:
Is it a checkpoint?

Reply:
It is unnecessary, as the device will be returned in the subsequent code